opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -77,6 +82,30 @@ export const objects = async (context: AuthContext, user: AuthUser, workspaceId:
   return listThings(context, user, types, finalArgs);
 };
 
+export const exportObjectsAsStixReportBundle = async (context: AuthContext, user: AuthUser, workspace: BasicStoreEntityWorkspace): Promise<string> => {
+  if (workspace.type !== 'investigation') {
```

Comment:
We don't have an enum to handle it ?

Reply:
I didn't find it, and I thought about creating one, but for now it's the only place we use it, so it didn't seem very useful to create an enum for one condition.